packages/@okta/vuepress-theme-default/enhanceApp.js

```diff
@@ -1,5 +1,7 @@
 import StackSelector from './components/StackSelector';
+import NextSectionLink from './components/NextSectionLink';
 
 export default ({ Vue }) => {
   Vue.component('StackSelector', StackSelector); // Force global register
+  Vue.component('NextSectionLink', NextSectionLink); // Force global register
```

Comment:
I missed this last time, but we should move the components that need to be global into the `global-components` folder as Vuepress does this for us automagically.

Reply:
I tried that and it didn't seem to work, but I'll give it another go.  I've got another effort I'm in the midst of to deal with some SSR issues, so if you are cool with it I'll merge this as is and move these in that PR (which makes sense because enhanceApp only runs client-side, so I have to move these to get SSR working.

Reply:
Sounds fine to me.